### PR TITLE
Issue 1274 import missing

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by doutriaux1** [comment]
goes with: https://github.com/UV-CDAT/uvcdat-testdata/pull/49

@remram44 @dlonie @chaosphere2112

File: Packages/vcs/Lib/Canvas.py [hunk 1]
```diff
@@ -885,8 +887,10 @@ def __init__(self, gui = 0, mode = 1, pause_time=0, call_from_gui=0, size=None,
 
         is_canvas = len(vcs.return_display_names()[0])
 
+        ## TODO get rid of all these gui_canvas_closed I think the code is not even here anymore
+        ## I believe it was for the old editor style
         if gui_canvas_closed == 1:
-           showerror( "Error Message to User", "There can only be one VCS Canvas GUI opened at any given time and the VCS Canvas GUI cannot operate with other VCS Canvases.")
+           raise RuntimeError("There can only be one VCS Canvas GUI opened at any given time and the VCS Canvas GUI cannot operate with other VCS Canvases.")
```
Comment (aashish24) [inline Packages/vcs/Lib/Canvas.py]: can we wrap this line?

**Comment by aashish24** [comment]
@doutriaux1 looks like more tests are failing now. 

@jbeezley does it not show +x (how many new tests are failing) in buildbot?

**Comment by jbeezley** [comment]
I'm not sure how the previous build is defined in CDash to make the comparison.  We may need to ask Ben or others in Kitware how to make that happen.

**Comment by doutriaux1** [comment]
@aashish24 they all passed for me, did you update uvcdat-testdata. Can you point to your cdash so I can take a look at what is missing.

**Comment by jbeezley** [comment]
You can get to the cdash page by clicking on "details" of the status below.

**Comment by doutriaux1** [comment]
ok that's interesting the 2 new tests are blank for you...

**Comment by jbeezley** [comment]
The baseline images don't exist for the new tests.

**Comment by doutriaux1** [comment]
ah! Well they will once the related bseline as been merged. That's probably something you want to work on your buildbot end. see: UV-CDAT/uvcdat-testdata#49

maybe you can query for link to PR on uvcdat-testdata to fetch this?

**Comment by jbeezley** [comment]
Alright, I'll have to learn more about how baselines are pulled from the other repo.

**Comment by jbeezley** [comment]
We will have to work out some sort of mechanism for automating this in the future.  For now, I think we will just have to validate baseline changes manually until the uvcdat-testdata repo is updated.

**Comment by aashish24** [comment]
Right. I think we have to fix the data problem regardless. I will talk to @jbeezley and @dlonie about it. My current thought is this: 

1) branch foo in uvcdat should have a branch foo in uvcdat-testdata repository
2) build should checkout both for testing purposes
3) integration testing (for future) should merge both of them into master and test

**Comment by dlonie** [comment]
The only issue I see with that approach is the race condition on the dashboards:

1) Dev pushes changes to code.
2) Buildbot checks out sources and (out-of-date/invalid/missing) testdata branch.
3) Dev pushes new baselines.

It's a training/workflow issue, but might be a headache.

**Comment by aashish24** [comment]
@dlonie that's a good point. I think this should be the order (could be enfocred by policy)

1) Push data first
2) Push code second 

I think thats not unreasonable since data push does not trigger a build. So some can push data and can push code anytime later. 

Let me know what you think of it so that I can add it to my documentation of UV-CDAT.

**Comment by dlonie** [comment]
Sounds good to me.

**Comment by doutriaux1** [comment]
@aashish24 @dlonie what's the hold on this?

**Comment by aashish24** [comment]
@doutriaux1 I think the bulidbot looks good (would have been nice to have a greener buildbot) but for now I am merging it.

**Pull request merged**